types/mdx/package.json

```
@@ -7,7 +7,7 @@
     "projects": [
         "https://github.com/mdx-js/mdx"
     ],
-    "tsconfigs": ["tsconfig.json", "tsconfig.preact.json", "tsconfig.react.json"],
+    "tsconfigs": ["tsconfig.minimal.json", "tsconfig.preact.json", "tsconfig.react.json"],
```

```suggestion
    "tsconfigs": ["tsconfig.frameworkless.json", "tsconfig.preact.json", "tsconfig.react.json"],
```

Based on the files in the PR?